src/options/default-node-options.ts

```diff
@@ -8,70 +8,70 @@ export const defaultNodeOption: RenderOption = {
         return ``
     },
     [NodeType.PARAGRAPH]:(node: Node, next: Next) => {
-         return `<p>${next(node.children)}</p>`
+        return `<p${addStyleAttrs(node.attrs.style)}>${next(node.children)}</p>`
```

Comment:
- Instead of providing style only, you can return attributes that contain almost all the data users can utilize.

- Please provide a space between the tag name and the function.

Reply:
1. The rest of the required attributes are handled separately. Its not handled in the same function because there are cases where attr name is different than that of attr key. For e.g. in case of src, sometimes we may get attr.url instead of attr.src. 
2. if there are no style attr, the output tag comes with a space. To handle this, there's no space between tag and string literal.

Reply:
What would be the case when a user wants to get support for custom attributes or predefined attributes? Have we also included support for this in the code?